omc3/knob_extractor.py

```diff
@@ -61,13 +61,30 @@
 
 
 """
+from __future__ import annotations
+
+####### WORKAROUND FOR JAVA ISSUES WITH LHCOP ##################################
+# Move `/mcr/bin` to the end of the Path.
+# Fixes:
+# 'Error: Could not find or load main class (...) aircompressor-0.26.jar'
+#
+# This ia a hack, please remove at the earliers convenience. For updates see: 
+# https://cern.service-now.com/service-portal?id=ticket&table=incident&n=INC3768823
+import os
+if "PATH" in os.environ and "/mcr/bin" in os.environ["PATH"]:
+    parts = os.environ["PATH"].split(":")
+    parts.remove("/mcr/bin")
+    parts.append("/mcr/bin")  # probably not neccesary
+    os.environ["PATH"] = ":".join(parts)
+################################################################################
```

Comment:
Cool that this works, annoying that we have to do it. Feel free to leave a `TODO` in there too, might be going under our radar after some time. Feel free to ignore too.